src/components/NestedForm/NestedFormInput.tsx

```diff
@@ -85,6 +93,25 @@ export class NestedFormInput extends React.PureComponent<data.IFieldInputProps,
 
         this.fieldStatus.error = error ? 'nestedError' : '';
     }
+
+    private renderCreateButton() {
+        const source = this.props.createButton;
+
+        if (!source || typeof source === 'string') {
+            return (
+                <button className='form-input-nested-button form-input-nested-create-button' type='button' onClick={this.onCreateEntry}>
+                    {source || 'Create'}
+                </button>
+            )
+        }
+
+        if (React.isValidElement(source)) {
+            return React.cloneElement(source, {
+                onClick: this.onCreateEntry,
+                className: classNames(source.props.className, 'form-input-nested-button form-input-nested-create-button'),
+            })
+        }
+    }
```

Comment:
Anyway we should return something, in order not to break the experience. Or throw err is fine.